install/pre-installation.md

```diff
@@ -45,26 +45,56 @@ Any steps run on an `external` server require that server to have the following
    >
    > - `-C -` is used to allow partial downloads. These tarballs are large; in the event of a connection disruption, the same `curl` command can be used to continue the disrupted download.
    > - **If air-gapped or behind a strict firewall**, then the tarball must be obtained from the medium delivered by Cray-HPE. For these cases, copy or download the tarball to the working
-   >   directory and then proceed to the next step. The tarball will need to be fetched with `scp` during the [Download CSM tarball](#21-download-csm-tarball) step.
+   > - directory and then proceed to the next step. The tarball will need to be fetched with `scp` during the [Download CSM tarball](#21-download-csm-tarball) step.
 
-   ```bash
-   # e.g. an alpha : CSM_RELEASE=1.3.0-alpha.99
-   # e.g. an RC    : CSM_RELEASE=1.3.0-rc.1
-   # e.g. a stable : CSM_RELEASE=1.3.0  
-   CSM_RELEASE=1.3.0-alpha.9
-   ```
+   1. (`external#`) Set the CSM RELEASE version
 
-   ```bash
-   curl -C - -f -O "https://artifactory.algol60.net/artifactory/csm-releases/csm/$(awk -F. '{print $1"."$2}' <<< ${CSM_RELEASE})/csm-${CSM_RELEASE}.tar.gz"
-   ```
+      ```bash
+      # e.g. an alpha : CSM_RELEASE=1.3.0-alpha.99
+      # e.g. an RC    : CSM_RELEASE=1.3.0-rc.1
+      # e.g. a stable : CSM_RELEASE=1.3.0  
+      CSM_RELEASE=1.3.0-alpha.9
+      ```
 
-1. (`external#`) Extract the LiveCD from the tarball.
+   1. (`external#`) Download the CSM tarball
 
-   ```bash
-   OUT_DIR="$(pwd)/csm-temp"
-   mkdir -pv "${OUT_DIR}"
-   tar -C "${OUT_DIR}" --wildcards --no-anchored --transform='s/.*\///' -xzvf "csm-${CSM_RELEASE}.tar.gz" 'cray-pre-install-toolkit-*.iso'
-   ```
+      > ***NOTE:*** CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
+      Using **http_proxy** or **https_proxy** in any way other than the following examples will cause many failures in subsequent steps.
+
+      Without proxy:
+
+      ```bash
+      curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
+        "https://artifactory.algol60.net/artifactory/csm-releases/csm/$(awk -F. '{print $1"."$2}' <<< ${CSM_RELEASE})/csm-${CSM_RELEASE}.tar.gz"
+      ```
+
+      With https proxy:
+
+      ```bash
+      https_proxy=https://example.proxy.net:443 curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
+        "https://artifactory.algol60.net/artifactory/csm-releases/csm/$(awk -F. '{print $1"."$2}' <<< ${CSM_RELEASE})/csm-${CSM_RELEASE}.tar.gz"
+      ```
+
+      With http proxy:
+
+      ```bash
+      http_proxy=https://example.proxy.net:80 curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
+        "https://artifactory.algol60.net/artifactory/csm-releases/csm/$(awk -F. '{print $1"."$2}' <<< ${CSM_RELEASE})/csm-${CSM_RELEASE}.tar.gz"
+      ```
+
+   - `scp` from the external server used in [Prepare installation environment server](#11-prepare-installation-environment-server):
```

Comment:
Based on the note above, I think the scp step is an OR step.  Either do the curl OR scp from the medium provided by HPE.

Reply:
Actually, I don't think the scp step belongs up here.   It looks like that is only needed below under "Download CSM tarball".

Reply:
It is a subsection of the Download CSM tarball, it was just rendering incorrectly.